src/main/content/_assets/css/home.scss

```diff
@@ -86,50 +78,27 @@ body{
     height:120px;
     background-color: #d8d8d8;
     color: #000000;
-    border:5px solid #000000;
```

Comment:
Border was specified in wireframe as 4px solid black.  My error in putting 5px, but I think border should be there.
<img width="1436" alt="Screen Shot 2019-09-05 at 9 07 57 AM" src="https://user-images.githubusercontent.com/37549026/64344563-b345ad00-cfbc-11e9-91c2-a30122c6acc8.png">